ci/azure-pipelines.yml

```diff
@@ -20,23 +20,13 @@ jobs:
     - script: |
         sudo apt-get install python python-dev python-crypto python-wand python3-pip python3-setuptools
         sudo apt-get install build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev wget libsqlite3-dev
-        pushd ..
-        wget https://www.python.org/ftp/python/3.7.3/Python-3.7.3.tar.xz
-        tar -xf Python-3.7.3.tar.xz
-
-        python3 --version
-        cd Python-3.7.3
-        ./configure
-        make build_all -j8
-        sudo make altinstall
-        popd
-        python3.7 --version
-
-        sudo python3.7 -m pip install --upgrade pip
-        sudo python3.7 -m pip install --upgrade setuptools
-        sudo python3.7 -m pip install mu_environment
```

Comment:
Are pip and setuptools upgraded as part of the UsePythonVersion task?  Is mu_environment installed also?

Reply:
No, will need to fix that.
Its frustrating there isn't a way to test the pipeline without pushing commits.

Reply:
Should we stand up a dev pipeline that is manually queued?

Reply:
Still doesn't matter, it can only run against commits that are pushed into the upstream repo.

Closest we can do it create a testing branch here we can push stuff into, but we would still need to do pull requests into that branch.

Reply:
I was referring to the latter. You can make changes, commit, and manually queue the pipeline to test it.

Reply:
Sure, I can do that in the future.